.github/workflows/build.yaml

```
@@ -35,6 +35,23 @@ jobs:
           path: build/dokka/htmlMultiModule/
           if-no-files-found: error
 
+  connected:
+    runs-on: ubuntu-latest
```

```suggestion
    runs-on: macos-latest
```
The emulator is not accelerated on Linux unless you pay for it. Mac is our only option. It at least has some free minutes until we start paying.

---

Ah gotcha, that explains why the CI build was taking forever. Thanks for all the help!